extensions/wikia/Scribe/ScribeEventProducerController.class.php

```diff
@@ -59,65 +40,19 @@ static public function onSaveComplete( &$oPage, &$oUser, $text, $summary, $minor
 		return true;
 	}
 
-	static public function onSaveRevisionComplete( $oPage, $oRevision, $revision_id, $oUser, $allow = true ) {
+	static public function onArticleDeleteComplete( &$oPage, &$oUser, $reason, $page_id ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused function parameter "$reason". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1172)